packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1507,7 +1514,7 @@ export class ContainerRuntime extends EventEmitter
                     sequenceNumber: message.sequenceNumber,
                     snapshot: attachMessage.snapshot ?? {
                         id: null,
-                        entries: [createAttributesBlob(pkg)],
+                        entries: [createAttributesBlob(pkg, false)],
```

Comment:
nit: I noticed Mark does the same, so I guess I'm not the only one who's missing better coding guidelines :)
Consider for arguments like to comment them, i,e,
```
createAttributesBlob(
   pkg,
   false, // isRoot
)]
```
or
```
createAttributesBlob(pkg, false /* isRoot */)]
```

Reply:
I like that!